components/badge.css

```diff
@@ -0,0 +1,42 @@
+
+.badge-padding-1 {
```

Comment:
Naming could be better. Using 1, 2 can be confusing.

Reply:
i am thinking of using numericals because its more convenient in case i have to add more variants in the future. So naming them large, x-large, xx-large would not look good. Also, i would be making the documentation site which will mention all this code.

Reply:
i have updated the naming. if it's fine shall i resolve the conversation?

Reply:
hey @sainath7878 can you please confirm if the changes are okay?